pull_request_template.md

```diff
@@ -11,20 +19,13 @@
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 Fixes # (issue)
 
-# Type of change
-
-- [ ] Bug fix (non-breaking change which fixes an issue)
-- [ ] New feature (non-breaking change which adds functionality)
-- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-- [ ] This change requires a documentation update
 
 # Checklist:
 
-- [ ] My code follows the style guidelines of this project
-- [ ] I have performed a self-review of my own code
+
+
 - [ ] I have commented my code, particularly in hard-to-understand areas
-- [ ] I have made corresponding changes to the documentation
 - [ ] My changes generate no new warnings
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing unit tests pass locally with my changes
-- [ ] Any dependent changes have been merged and published in downstream modules
+- [ ] Any dependent changes have been merged and published in downstream modules
```

Comment:
I would suggest the following checklist for your project:
- rubocop doesn't raise any warnings and offenses
- all specs pass 
- new features are covered with new specs
- PR is approved by 2 team mates
- PR is approved by 2 experts

@okohuch what do you think?

Reply:
@vikabelous I like the list you suggested.
@imysheniuk update the checklist according to Vika's comment